F21/annoyed-raven/main.py

```diff
@@ -0,0 +1,104 @@
+import csv
+
+capitalized = []
+lowercased = []
+language_encoding = []
+
+'''
+This function opens the tsv file and seperates tests cases into three parts
+the language encoding, Capitilazed input, and the suggested Lowercased result
+'''
+with open("tests.tsv", encoding="UTF-8") as language_tsv:
+    language_tsv = csv.reader(language_tsv, delimiter="\t", quotechar='"')
+    for row in language_tsv:
+        transition = []
+        for x in row:
+            transition.append(x)
+         
+        capitalized.append(transition[0])
+        language_encoding.append(transition[1])
+        lowercased.append(transition[2])       
+'''     an easy check to make sure tsv file was seperated correctly
+        print(row)
+        print(capitalized)
+        print(language_encoding)
+        print(lowercased)
+'''  
+# the storage lists of indexes for each occurence of a the specific dialact/language
+Encoding_indexesEN= []
```

Comment:
A 2D list or dictionary may be beneficial here as it would allow for fewer variables and could reduce the amount of for loops further down in the file when converting to lowercase. A dictionary may be more useful as it would allow for using the same variable names within the overall variable. It would also make it easier for the program to handle additional languages being added to the test file